backend/kesaseteli/applications/templates/youth_summer_voucher_email.html

```diff
@@ -64,8 +64,8 @@ <h2>{% trans 'Ohjeita työnhakijalle' %}</h2>
                     <h2>{% trans 'Ohjeita työnantajalle' %}</h2>
                     <p>
                         {% trans 'Kesäseteli on nuorten, yrittäjien ja Helsingin kaupungin yhteinen hanke.' %}
-                        {% trans 'Kaupunki korvaa 325 euroa työnantajalle, joka palkkaa Kesäseteliin oikeutetun nuoren töihin 1.6.–15.8.2023 välisenä aikana.' %}
-                        {% trans 'Kesätyön vähimmäisvaatimuksina ovat 60 tunnin työaika ja 400 euron palkka.' %}
+                        {% trans 'Kaupunki korvaa 350 euroa työnantajalle, joka palkkaa Kesäseteliin oikeutetun nuoren töihin 1.6.–15.8.2024 välisenä aikana.' %}
```

Comment:
Instead of using hardcoded values in hard text, variables should be used. Then we could have a constant price catalog and the next update would be even easier. 

Here is how to use the variables: 
1. https://docs.djangoproject.com/en/5.0/topics/i18n/translation/#standard-translation
2. https://docs.djangoproject.com/en/5.0/topics/i18n/translation/#blocktranslate-template-tag

Reply:
I thought about it, but didn't implement it now. It is a good idea for refactoring and maintainability, but not something I will implement in this PR. Let's see if I make another PR for refactoring this or not...